txn/flusher.go

```diff
@@ -921,6 +921,10 @@ func (f *flusher) apply(t *transaction, pull map[bson.ObjectId]*transaction) err
 			}
 		case op.Assert != nil:
 			// Pure assertion. No changes to apply.
+			if f.opts.AssertionCleanupLength > 0 && len(pullAll) >= f.opts.AssertionCleanupLength {
```

Comment:
It is a nitpick, but the comment here is misleading. It might cause confusion if someone looks at it in half a year without prior knowledge of this PR and how the transactions handling have changed. 

Could you please remove/update it?

Reply:
great catch, will do